python/pyspark/sql/session.py

```diff
@@ -252,6 +252,16 @@ def newSession(self):
         """
         return self.__class__(self._sc, self._jsparkSession.newSession())
 
+    @since(2.4)
+    def getActiveSession(self):
+        """
+        Returns the active SparkSession for the current thread, returned by the builder.
+        >>> s = spark.getActiveSession()
+        >>> spark._jsparkSession.getDefaultSession().get().equals(s.get())
```

Comment:
So normally we try and have doc tests like these be examples of how the user should use this. So I would consider getting the active session and then doing something a normal user would with it (like paralleling a collection).

Reply:
..and probably shouldn't access `_jsparkSession`

Reply:
@holdenk @felixcheung Thanks for the review. I will change this.